test/runtests.jl

```diff
@@ -16,20 +19,28 @@ using Base.Test
 ipopt_solver = IpoptSolver(tol=1e-6, print_level=0)
 
 # this will work because PowerModels is a dependency
+pmpath = Pkg.dir("PowerModels") # doing like this so that it does not depend on relative paths
```

Comment:
Why would this be considered unstable? I would think this should be more general (the reason why I changed this is that the current hardcoded paths do not work on my setup). There is also the added gain of using `join` to compose the path, which works wether the system is Unix or not.

Reply:
details here, https://github.com/JuliaLang/METADATA.jl/pull/7827

Reply:
Thanks for the link, now I see your point. I would still argue that the same instability exists with the hardcoded paths (PowerModels and PowerModelsAnnex may be in completely different folders, if you are appending to `LOAD_PATH`). Anyway, I am removing this part from the PR, since it is not its focus and bringing back the hardcoded paths.